Makefile

```diff
@@ -44,25 +17,24 @@ install: ## install Poetry, which in turn installs the dependencies and makes an
 	poetry install
 
 .PHONY: test
-test: dsp-stack ## run all tests located in the "test" folder (intended for local usage)
-	-poetry run pytest test/	# ignore errors, continue anyway with stack-down (see https://www.gnu.org/software/make/manual/make.html#Errors)
-	$(MAKE) stack-down
+test: ## run all tests located in the "test" folder
+	poetry run dsp-tools start-stack --no-prune
+	-poetry run pytest test/	# ignore errors, continue anyway with stop-stack (see https://www.gnu.org/software/make/manual/make.html#Errors)
+	poetry run dsp-tools stop-stack
```

Comment:
you could also look into the [poetry exec plugin](https://github.com/keattang/poetry-exec-plugin) to automate poetry workflows, which might spare you even more of the makefile

Reply:
Good idea, it would be cool to get rid of the Makefile, esp. because there are only 2 commands left in it, each a 1-liner. But I'm a bit reluctant using a plugin which is still pre-1.0... What do you think?

Reply:
I think it would be worth trying, even if it's only to see how well it works. In any case, it would be easy enough to revert back to having the make file (and if it's only two one-liners, you could just as well put them into the readme and execute them manually).  
Speaking of those two commands: in the `make clean` command, are most of them still needed? I would guess that both mkdocs (for the docs) and poetry (for building) take care of removing old files pretty well.

Reply:
OK, done :-) 
There is only one exec-script remaining, and that one is only used by me. This allowed me to keep the documentation of the poetry-exec-plugin minimal.